README.md

```diff
@@ -53,8 +53,9 @@ learn more [here](https://github.com/ember-template-imports/ember-template-impor
 module.exports = {
   overrides: [
     {
-      files: ['**/*.{js,ts,gjs,gts}'],
+      files: ['**/*.{js,ts}'],
       plugins: ['ember'],
+      parser: '@typescript-eslint/parser',
```

Comment:
Shouldn't TS parser only be applied to TS files?

Reply:
ye --- its not what the ember blueprint does --- but it should, imo (more cases for overrides only)

Reply:
also, the example on typescript-eslint shows it  on top level...

Reply:
Not great